client/app/bundles/buttons/ButtonFilled.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import css from './button.scss';
+
+class ButtonFilled extends React.Component {
+   render() {
+       
+       
+      return (
+         <button className = {css.buttonFilled}>
+            Learn More
```

Comment:
Let's pass the button text as props e.g. {this.props.text}

Reply:
If using props, usage would look like this:
```
<ButtonFilled text="Learn More"/>
<ButtonFilled text="Learn More" icon={<Icon type="smiley"/>}/>
```
but if using children, usage would look like this
```
<ButtonFilled>
  Learn More
</ButtonFilled>

<ButtonFilled>
  <Icon type="smiley"/> Learn More
</ButtonFilled>
```